docs/docs/prompt-management/03-prompt-management-sdk.mdx

```diff
@@ -418,18 +454,31 @@ List of variants:
 
 ## Fetching a Variant's history
 
-To list all versions for a variant of an application, use the `VariantManager.list_variants` method.
+To list all versions for a variant of an application, use the `VariantManager.list` method.
 
 ```python
-# List all variants
-variants = ag.VariantManager.list_variants(
+# List all variant versions/history (synchronously)
+versions = ag.VariantManager.history(
+    variant_slug="variant-slug",
     app_slug="my-app"
     # app_id="my-app-id",  # you can also use `app_id`
 )
 
-print("List of variants:")
-for variant in variants:
-    print(variant)
+print("History of variant:")
```

Comment:
You have changed the print but did not change the output (still reading List of variants).

Reply:
Thank you!